cmd/contour/contour.go

```diff
@@ -163,6 +168,14 @@ func main() {
 		log.Infof("args: %v", args)
 		var g workgroup.Group
 
+		// Functionality warnings for non-functional flags.
+		if externalHTTPPort != 0 {
+			log.Infof("Due to changes to upstream Envoy, setting envoy-external-http-port to %d no longer does anything, your Ingress will work without it. This notice will be removed in two releases.", externalHTTPPort)
```

Comment:
shorter is better

log.Warn("--envoy-external-http-port will be removed in 0.14")